### ISSUE TITLE
Screen reader is not announcing 'Name', 'Role' and 'State' information for 'Menu', 'Stickers' and 'Suggestion' controls.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27873]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: [Telegram: Contact @SCTejTestbot](https://t.me/SCTejTestbot) on edge.
3. SCTestbot home screen will appear.
4. Activate on start button.
5. Navigate to Stickers control in edit field using tab key.
6. Turn on Narrator.
7. Try to navigate to the  controls using 'Tab' key.
8. Observe an issue that Screen reader is not announcing 'Name', 'Role' and 'State' information for 'Menu', 'Stickers' and 'Suggestion' controls.

### Expected behaviour

Screen reader should announce 'Name', 'Role' and 'State' information for 'Menu', 'Stickers' and 'Suggestion' controls.
Observation: While navigating with 'Tab' key, screen reader focus lands on info control and it should announce name as 'More options', role as 'Button' and state as collapses/expanded.

### Actual behaviour

Actual:
Screen reader is not announcing 'Name', 'Role' and 'State' information for 'Menu', 'Stickers' and 'Suggestion' controls.
Observation: While navigating with 'Tab' key, screen reader focus lands on more control and it is announcing as group, here name and role of the more control is not announced by screen reader.

https://github.com/telegramdesktop/tdesktop/assets/93735775/ca24ae2d-9ab5-4f65-984e-065185db06b0

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476

can you stop posting issues for screen reader and keyboard only things? i have closed you a mountain of duplicate issues.

**Comment by msftedad** [comment]
Hi @Aokromes, Yesterday we have logged 16 issues related to telegram and we noticed that all issues are closed as completed by you. We have a bunch of telegram issues, could you please guide us on logging these issue?

**Comment by john-preston** [comment]
@msftedad

The problem is, all those issues don't need to be described in detail in regards to the screen reader support, there is a single issue requesting screen reader support in general, which is not in TDesktop, so no sense in creating different issues about separate parts of interface that don't support screen reader - none do.

The same comes to the keyboard control of some controls - almost all controls, like buttons etc, don't have a feature of keyboard focus and so you can't jump to them using keyboard. No need to create a thousand of issue with different button examples, they all don't support it.

Similarly with text-to-backgroune contrast. The design for TDesktop is provided by the company and is not discussed in this repository. You can create issue on https://bugs.telegram.org about that but I doubt they'll be addressed at all.

**Comment by msftedad** [comment]
Hi @john-preston, we have below 37 Telegram issues apart from text-to-background contrast issues. Shall we log these on single issue or will you merge these issues. Kindly let us know for further action.
1.	By using keyboard focus moving to hidden after 'microphone' control.
2.	Controls present in header section of the chat are not accessible using keyboard.
3.	Dropdown control is not accessible using swipe gesture.
4.	Expanded 'menu' control is not collapse with esc key.
5.	Focus is not moving to ‘Back’ icon.
6.	Heading level is not defined for ‘SCTestBot’ text.
7.	Keyboard focus is stuck in input field while navigating using tab key.
8.	Large text is not adapted in telegram application.
9.	Links present in 'received message' section is not accessible by using swipe gesture.
10.	Name and role are not defined for ‘Share’ control.
11.	Numbers are not showing for edit field.
12.	'Quick Reactions' controls are not accessible by using keyboard.
13.	'Received and send message' are not accessible by using keyboard.
14.	Role is not defined for ‘Search’ and ‘Share’ controls in ‘Share’ popup.
15.	Role is not defined for Quick reactions controls.
16.	Screen reader focus is not landing on ‘Menu’ controls after activating ‘Menu’ control.
17.	Screen reader focus is not logical on 'See more on Bing' control by using swipe gesture.
18.	Screen reader focus is not logical on 'start' control by using swipe gesture after 'more option' control.
19.	Screen reader focus is not logical 'zero input' page.
20.	Screen reader focus is not moving to interactive controls in ‘Share’ popup.
21.	Screen reader focus not landing on 'received message' info after invoking on suggestion control.
22.	Screen reader is announcing unnecessary information as 'disabled' for link control.
23.	Screen reader is not announcing ‘Start’ control tooltip information.
24.	Screen reader is not announcing ‘The conversation has reached its limit’ information once chat page is reached its limit.
25.	Screen reader is not announcing ‘The conversation has reached its limit’ information once chat page is reached its limit.
26.	Screen reader is not announcing any information about ‘See More on Bing’ popup after activating that control.
27.	Screen reader is not announcing any information after activating ‘Start’ control.
28.	Screen reader is not announcing role for 'profile' control.
29.	Screen reader not announcing expanded/collapse 'state' for 'more option' control.
30.	Screen reader not announcing heading level 'what can this bot do?' text.
31.	Screen reader not announcing 'received message' information after invoking on suggestion control.
32.	Screen reader not announcing role for expanded more option list controls.
33.	Screen reader not announcing 'status' information about 'bot typing'.
34.	Screen reader not announcing status message 'bot typing' after sending any query to the bot.
35.	'See more on Bing' control is not accessible by using tab key.
36.	Unnecessary numbers are showing for non-interactive controls.
37. Visible and accessible name is different for 'Saved messages' control.

**Comment by msftedad** [comment]
Hi @john-preston, Could you please confirm if these issues are in your consideration, or shall we log them separately?

**Comment by Aokromes** [comment]
> Hi @john-preston, Could you please confirm if these issues are in your consideration, or shall we log them separately?

if you open 37 issues for screen reader things i will close them as duplicate of https://github.com/telegramdesktop/tdesktop/issues/476
